templates/controls/import_modal.html

```diff
@@ -0,0 +1,34 @@
+<div id="project_modal" class="modal" tabindex="-1" role="dialog" aria-labelledby="project_modal_title" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+        <h4 class="modal-title" id="project_modal_title">Select a Project</h4>
+      </div>
+<form method="post" enctype="multipart/form-data" action="{% url 'project_import' project.id %}">
```

Comment:
How does the import project modal have access to a project id before the project is imported? What project is this id coming from?

Reply:
Right so in the modal we allow the user to choose to make a new project (with the checkbox) or update the current one. The latter is allowed with this project.id and if they want to create a new project we need the system and portfolio information from the current project we are on.